test/integration/integration_test.cc

```diff
@@ -560,7 +560,8 @@ TEST_P(IntegrationTest, Http10WithHostandKeepAliveAndLws) {
                                 "GET / HTTP/1.0\r\nHost: foo.com \r\nConnection:Keep-alive\r\n\r\n",
                                 &response, true);
   EXPECT_THAT(response, HasSubstr("HTTP/1.0 200 OK\r\n"));
-  EXPECT_THAT(response, Not(HasSubstr("connection: close")));
```

Comment:
This removal reflects the fact we're now invoking the HCM drain path, which is all to the good.
We could tweak the HCM drain path to not append connection: close for HTTP/1.0 (since it's implicit) but honestly I prefer the increased clarity and minimized complexity at the cost of those extra bytes on the wire.

Reply:
nit: maybe explicitly test for presence of the header?